storage/store.go

```diff
@@ -79,6 +80,10 @@ var (
 	}
 )
 
+func init() {
+	gob.Register(&NodeDescriptor{})
```

Comment:
Let's locate this statement in range.go, along with the others related to gossip.